style.css

```diff
@@ -0,0 +1,137 @@
+* {
+	box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+  }
+
+html {
+font-size: 14px;
+color: rgb(27, 27, 27);
+}
+
+body {
+font-family: 'Roboto', sans-serif;
+background-color: #eceef1;
+}
+
+
+header {
+height: 60px;
+width: 100%;
+
+z-index: 999;
+margin-right: 40px;
+
+}
+.header-container{
+	display:flex;
+	justify-content: flex-end;
+	padding:10px 30px;
+}
+
+  a {
+
+	text-decoration: none;
+	color:#0077C5;
+  }
+
+.lock{
+	background-image: url(images/icon-whitelock@3x.png);
+	display:inline-block;
+	height: 1em;
+	width:.8em;
+	background-size: contain;
+	background-repeat: no-repeat;
+	margin-top: 2px;
+    margin-right: 4px;
+    vertical-align: text-top;
+}
+.signin-container{
+	margin-left:20px;
+}
+.signin-btn{
+	background: #0077C5;
+    border-radius: 4px;
+	border: 1px solid #4c4c4c;
+	height: 34px;
+	padding: 0 20px;
+    color: white;
+    cursor: pointer;
+    font-size: 1.125em;
+    font-weight: normal;
+}
+main {
+	display: flex;
+	justify-content: center;
+
+  text-align: center;
+	/*background-color: DodgerBlue;*/
+  }
+
+main>div {
+	width: 400px;
+
+  }
+label{
+	display: block;
+
+}
+.form-content{
+	text-align: left;
+  padding: 10px;
+}
+input{
+	width: 370px;
+	height: 40px;
+}
+.create-btn{
+	background: #0077C5;
+  border-radius: 4px;
+	border: 1px solid #4c4c4c;
+	height: 34px;
+	padding: 0 20px;
+  color: white;
+  cursor: pointer;
+  font-size: 1.125em;
+  font-weight: normal;
+	width: 370px;
+	height: 50px;
+
+}
+.terms-condition{
+	color:grey;
+	padding: 10px;
+	font-style: italic;
+}
+
+.form-body{
+	background-color: #ffffff;
+  border: 1px solid rgb(199, 199, 199);
+}
+.recaptcha{
+	background-color: #eceef1;
+	padding: 10px;
+}
+
+footer{
+	text-align: center;
+  margin-top: 10px;
+}
+.copyright{
+	margin-left: 200px;
+	margin-right: 200px;
+	margin-top: 10px;
+	margin-bottom: 10px;
```

Comment:
Lines 121, 122, 123 and 124 can be reduced to one line. margin: 10px 200px;

Reply:
so can we go ahead and merge the pull request or we have to submit a new pull request

Reply:
@JamesOkunlade   so can we go ahead and merge the pull request or we have to submit a code review after making that change

Reply:
OK got it @JamesOkunlade